lms/resources/_js_config/__init__.py

```diff
@@ -15,7 +15,7 @@ class JSConfig:
     class Mode(str, Enum):
         OAUTH2_REDIRECT_ERROR = "oauth2-redirect-error"
         BASIC_LTI_LAUNCH = "basic-lti-launch"
-        CONTENT_ITEM_SELECTION = "content-item-selection"
+        FILE_PICKER = "content-item-selection"
```

Comment:
I'll change the FE side of this in other PR, there it seems this is the only wrinkle and the "FilePicker" naming is the preferred one.

I rather do this after tackling LTI1.1 which involves a lot (for my FE ability) FE changes, we do use a enum after all.